.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  tox:
+    uses: ./.github/workflows/tox.yml
+    secrets: inherit
+  publish:
+    needs: [Tox]
```

Comment:
Job names are case sensitive, would rename it to lowercase (`tox`).

Reply:
done